packages/medusa/src/services/sales-channel-inventory.ts

```diff
@@ -40,14 +40,13 @@ class SalesChannelInventoryService {
     salesChannelId: string,
     inventoryItemId: string
   ): Promise<number> {
-    const locations = await this.salesChannelLocationService_.listLocations(
-      salesChannelId
-    )
-
-    return await this.inventoryService_.retrieveAvailableQuantity(
-      inventoryItemId,
-      locations
-    )
+    const locations = await this.salesChannelLocationService_
```

Comment:
self-review: @carlos-r-l-rodrigues here the transaction was not passed along for those calls, plus this service was not supporting the transactions. I did some changes, can you let me know what you think

Reply:
looks fine. thanks